pypesto/profile/options.py

```diff
@@ -112,5 +112,5 @@ def validate(self):
         if self.default_step_size < self.min_step_size:
             raise ValueError("default_step_size must be >= min_step_size.")
 
-        if self.magic_factor_obj_value < 0 or self.magic_factor_obj_value >= 1:
-            raise ValueError("magic_factor_obj_value must be >= 0 and < 1.")
+        if self.magic_factor_obj_value < 1:
+            raise ValueError("magic_factor_obj_value must be > 1.")
```

Comment:
The magic factor now has to be larger than 1 rather than between 0 and 1. The next_obj_target now depends on the objective function value difference of the last two profile points, rather than the difference of the last profile point to the global minimum. The former is in general much smaller than the latter, so we want to increase it rather than decrease it using a factor.

Reply:
True...
It is kinda the scaling factor of the adaptive target value.
Changed to `adaptive_target_scaling_factor`.

Reply:
Take a look at the documentation of it in ProfileOptions. Not sure if informative enough.